scripts/rabbitmqctl.bat

```diff
@@ -60,4 +62,3 @@ if ERRORLEVEL 1 (
 EXIT /B 0
 
 endlocal
-endlocal

```

Comment:
There is two `setlocal` at the beginning of the script. I see [this paragraph on ss64.com](https://ss64.com/nt/setlocal.html):

> SETLOCAL can be used more than once in the same batch file so that multiple values can be stored in the same Environment Variable. To keep track of variable definitions, pair each SETLOCAL with a corresponding ENDLOCAL.

Shouldn't we keep the second `endlocal`?

Reply:
@lukebakken

Reply:
I must have missed the double setlocal statements.